src/Notes/Note.php

```diff
@@ -616,26 +616,23 @@ public function set_is_read( $is_read ) {
 	/**
 	 * Add an action to the note
 	 *
-	 * @param string  $name           Action name (not presented to user).
-	 * @param string  $label          Action label (presented as button label).
-	 * @param string  $url            Action URL, if navigation needed. Optional.
-	 * @param string  $status         Status to transition parent Note to upon click. Defaults to 'actioned'.
-	 * @param boolean $primary        Whether or not this is the primary action. Defaults to false.
-	 * @param string  $actioned_text The label to display after the note has been actioned but before it is dismissed in the UI.
+	 * @param string $name           Action name (not presented to user).
+	 * @param string $label          Action label (presented as button label).
+	 * @param string $url            Action URL, if navigation needed. Optional.
+	 * @param string $status         Status to transition parent Note to upon click. Defaults to 'actioned'.
+	 * @param string $actioned_text The label to display after the note has been actioned but before it is dismissed in the UI.
 	 */
 	public function add_action(
 		$name,
 		$label,
 		$url = '',
 		$status = self::E_WC_ADMIN_NOTE_ACTIONED,
-		$primary = false,
 		$actioned_text = ''
 	) {
```

Comment:
@chihsuan For backwards compatibility, I think we'll need to keep the `$primary` parameter in place, but just never use it and say that it is deprecated in the inline documentation.

There may be extensions and third party code in the wild using the previous method signature.

Reply:
> For backwards compatibility, I think we'll need to keep the $primary parameter in place, but just never use it and say that it is deprecated in the inline documentation.

@adrianduffell Okay, I'll create another PR to add this back today. Thanks!

Reply:
Sounds good, thanks! 👍